<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteFileDlg</class>
 <widget class="QDialog" name="DeleteFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete File</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../hla.qrc">
    <normaloff>:/png/images/about.png</normaloff>:/png/images/about.png</iconset>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>3</y>
     <width>591</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>check all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../hla.qrc"/>
 </resources>
 <connections/>
</ui>
